ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/js/dropdownview.js

```diff
@@ -84,7 +84,6 @@
         }
 
         updateEnabled(enabled, state) {
-            this.toggle(enabled, FormView.Constants.ARIA_DISABLED, true);
```

Comment:
Why is this code removed ? Isn't this backward incompatible

Reply:
This line was adding aria-disabled attribute to the parent div which is of no use, so I has removed this line.